internal/cli/docker_prune.go

```diff
@@ -50,9 +55,9 @@ func (c *dockerPruneCmd) run(ctx context.Context, args []string) error {
 	a.Incr("cmd.dockerPrune", nil)
 	defer a.Flush(time.Second)
 
-	// (Most relevant output from dockerpruner is at the `debug` level)
-	l := logger.NewLogger(logger.DebugLvl, os.Stdout)
-	ctx = logger.WithLogger(ctx, l)
+	// // (Most relevant output from dockerpruner is at the `debug` level)
+	// l := logger.NewLogger(logger.DebugLvl, os.Stdout)
```

Comment:
?

was this intentional?

Reply:
Fixed in 6404e57e2, logic now is:
* `tilt docker-prune` forces a _minimum_ of VERBOSE for logger (won't override `--debug` if passed)
* Pruner only writes out messages when _nothing_ for {cache,container,image} pruned (@ info lvl) if verbose enabled (previously was debug - slight behavior change here but aligns better since these are user messages and not Tilt internals)
* `tilt docker-prune` has NEW logging @ DEBUG which shows the resolved images it's going to prune on, this isn't always output because registry rewriting might confuse some people, but it can be useful extra context if you think it's not pruning something it should be

(That sounds much more complicated than it really is 😛 )